src/shared/util.js

```diff
@@ -1329,9 +1336,221 @@ MessageHandler.prototype = {
     return capability.promise;
   },
   /**
+   * Sends a message to the comObj to invoke the action with the supplied data.
+   * Expect that the other side will callback to signal 'start_complete'.
+   * @param {String} actionName - Action to call.
+   * @param {JSON} data - JSON data to send.
+   * @param {Object} queueingStrategy - strategy to signal backpressure based on
+   *                 internal queue.
+   * @param {Array} [transfers] - Optional list of transfers/ArrayBuffers.
+   * @return {ReadableStream} ReadableStream to read data in chunks.
+   */
+  sendWithStream(actionName, data, queueingStrategy, transfers) {
+    let streamId = this.streamId++;
+    let that = this;
+    let sourceName = this.sourceName;
+    let targetName = this.targetName;
+    return new ReadableStream({
+      start(controller) {
```

Comment:
`start: (controller) => {`, and similar below, should work and removes the need for binding `this` manually.